eternalcore-core/src/main/java/com/eternalcode/core/configuration/implementation/PluginConfiguration.java

```diff
@@ -306,6 +306,10 @@ public static class Format {
 
     @Contextual
     public static class Afk implements AfkSettings {
+
+        @Description("# Mark player as AFK when inactive?")
+        public boolean markPlayerAsAfk = true;
```

Comment:
Przenieś to nad "czas inactive" i może zmień nazwę na "autoInactiveAfk"
czy jakoś tak. (użyj słowa inactive bo to jest używane do tego ficzera, a Nadal można oznaczyć gracza komendą)